api/auth.js

```
@@ -0,0 +1,180 @@
+}
+
+function validateAuthToken(token) {
+  try {
+    const parts = token.split('.');
+    if (parts.length !== 3) return null;
+    const expectedSig = crypto
+      .createHmac('sha256', LICENSE_SECRET)
+      .update(`${parts[0]}.${parts[1]}`)
+      .digest('base64url');
+    if (!crypto.timingSafeEqual(
+      Buffer.from(expectedSig, 'base64url'),
```

ESLint

[error] 'Buffer' is not defined.

(no-undef)